### PR TITLE
Switch from GUID to UUID

### PR DESCRIPTION
**Pull request by kriswest**
@mattjamieson suggested we switch from GUID to UUID (as the more standard term). This PR update the bridging proposal PR to make that switch.

Hence `requestGuid` and `responseGuid` become `requestUuid` and `responseUuid` throughout the schemas.

(I also reapplied markdown linting to docs as we switched the chosen bullet style recently)

@Vivek-NatWest @tpina 

Please note I'll be working on a further PR to split off separate schemas for request/responses to/from the bridge (so that there is a separate schema for what the DA sends to the bridge and what it then forwards on, after adding some fields). I'll base it on this one on the assumption that this chan get is going ahead.

### CODE REVIEW

File: docs/agent-bridging/spec.md [hunk 1]
```diff
@@ -26,11 +26,11 @@ It is clear, however, that user desktops are substantially more complicated in r
 
 ![A single desktop and Desktop Agent, with external apps](/assets/dab-overview-2.png)
 
-Firms that make extensive use of FDC3 have also identified use cases where interoperability workflows span different physical desktops, and have built proprietary bridging solutions (e.g. [Backplane](https://backplane.finos.org/) was originally developed as an in-house solution to this problem).
+Firms that make extensive use of FDC3 have also identified use cases where interoperability workflows span different physical desktops, and have built proprietary bridging solutions (e.g. [BackPlane](https://backplane.finos.org/) was originally developed as an in-house solution to this problem).
```
[the remark this answers is not in the document]
Comment (kriswest) [reply docs/agent-bridging/spec.md:29]: think you're correct, reverting that

Comment (kriswest) [inline docs/agent-bridging/spec.md:29]:
```suggestion
Firms that make extensive use of FDC3 have also identified use cases where interoperability workflows span different physical desktops, and have built proprietary bridging solutions (e.g. [Backplane](https://backplane.finos.org/) was originally developed as an in-house solution to this problem).
```

**Comment by kriswest** [comment]
@mhmcclung making sure you are aware of this change (see PR). Will merge it later on today I hope.

There is a further one to come for the schemas (separating messages from DA -> Bridge from Bridge -> DA - content of messages won't change, but schemas should be more precise (right now one schema covers both messages, where they should be more specific).

**Comment by kriswest** [comment]
@Vivek-NatWest Wanted to make sure you are aware of this change also before we merge it.

**Review by tpina — APPROVED** [review]

**Pull request merged**